### ISSUE TITLE
Ledger can't send

### ISSUE THREAD

**Issue by zillionn** [ArkEcosystem/desktop-wallet#634]
**Describe the bug**
`Send` button doesn't appear with Ledger accounts. Also total amount is `0`.

**Screenshots**
<img width="955" alt="screenshot 2018-12-04 at 00 12 47" src="https://user-images.githubusercontent.com/3462869/49404910-888ea580-f759-11e8-8a8f-bb2b8ae54846.png">

**Server (please complete the following information):**
 - OS: Mac OS
 - Version: Mojave 10.14.1

**Comment by sleepdefic1t** [comment]
Can confirm issue.

`send` seems to only appear on new Ledger addresses without sent tx's.

Balances also show `0` in wallet-view, as well when Ledger address is added in "contacts".

- OS: Mac OS
- Version: Mojave 10.14.1

**Comment by sleepdefic1t** [comment]
`0` Balance issue likely the same causing #632.

**Comment by reConNico** [comment]
Can confirm `send` problem as well. For Ledger addresses without tx's the button appears.

* OS: Ubuntu
* Version: 18.04

**Comment by alexbarnsley** [comment]
🤔 	the only thing that stops the "Send" button showing is if the wallet is not a contact. I presume you've all not added the wallet as a contact. Can someone show the the Console tab in the devtools please, ignoring any 404 errors (core v2 returns 404 error if wallet doesn't exist on the chain)

**Comment by zillionn** [comment]
The only error I got.

<img width="1280" alt="screenshot 2018-12-04 at 18 52 37" src="https://user-images.githubusercontent.com/3462869/49458711-0a341100-f7f6-11e8-80cd-8263172fc147.png">

**Comment by zillionn** [comment]
Oh, actually I have my Ledger wallets as contacts!

**Comment by zillionn** [comment]
So, removed one of the wallets from the contacts and it showed the Send button.

OK, what's the best way to monitor my Ledger wallets when the Ledger is not connected? I've added them as contacts so I can see their txs.

So, I tried to add one of my Ledger wallets to `My Wallets` not to `Contacts`:
1. Disconnected the Ledger
2. Added one of my Ledger wallets `Ledger 2` to `My Wallets` and labeled it `Ledger 22222`
3. Connected the Ledger
4. In `My Wallets` I can see the wallet twice but both with label `Ledger 2`

**Comment by reConNico** [comment]
>  the only thing that stops the "Send" button showing is if the wallet is not a contact. I presume you've all not added the wallet as a contact. Can someone show the the Console tab in the devtools please, ignoring any 404 errors (core v2 returns 404 error if wallet doesn't exist on the chain)

I actually added my first ledger address as contact before - so that seems like the problem

**Comment by alexbarnsley** [comment]
I will be fixing this so if it's a ledger, it disregards any other type of wallet stored. So if the ledger is a contact, it won't use that "wallet", but the actual ledger wallet from the device. Don't know if that makes sense 😂

**TL;DR** me fix so send shows even if added as contact too
